src/rules/prefer-reactive-destructuring.ts

```diff
@@ -0,0 +1,67 @@
+import type { TSESTree } from "@typescript-eslint/types"
+import { createRule } from "../utils"
+
+export default createRule("prefer-reactive-destructuring", {
+  meta: {
+    docs: {
+      description: "Prefer destructuring from objects in reactive statements",
```

Comment:
The first letter should be lowercase to be consistent with existing rules. (see #218)